site/public/js/grade-inquiry.js

```diff
@@ -30,7 +50,6 @@ function onReady() {
 
 function onComponentTabClicked(tab) {
     const component_id = $(tab).data('component_id');
-
```

Comment:
This whitespace change should be reverted

Reply:
Hi William, I removed the whitespace

Reply:
It looks like a line is still being deleted here